ui/admin/tests/acceptance/users/update-test.js

```diff
@@ -27,52 +29,62 @@ module('Acceptance | users | update', function (hooks) {
   };
 
   hooks.beforeEach(function () {
-    orgScope = this.server.create(
-      'scope',
-      {
-        type: 'org',
-      },
-      'withChildren'
-    );
-
+    instances.scopes.global = this.server.create('scope', { id: 'global' });
+    instances.scopes.org = this.server.create('scope', {
+      type: 'org',
+      scope: { id: 'global', type: 'global' },
+    });
     instances.user = this.server.create('user', {
-      scope: orgScope,
+      scope: instances.scopes.org,
     });
-
-    usersURL = `/scopes/${orgScope.id}/users`;
-    userURL = `${usersURL}/${instances.user.id}`;
+    urls.orgScope = `/scopes/${instances.scopes.org.id}`;
+    urls.users = `${urls.orgScope}/users`;
+    urls.user = `${urls.users}/${instances.user.id}`;
 
     authenticateSession({});
   });
+
   test('can save changes to an existing user', async function (assert) {
     assert.expect(2);
-    await visit(userURL);
+    await visit(urls.users);
+
+    await click(`[href="${urls.user}"]`);
     await click('form [type="button"]', 'Activate edit mode');
     await fillIn('[name="name"]', 'Updated user name');
     await click('.rose-form-actions [type="submit"]');
-    assert.strictEqual(currentURL(), userURL);
+
+    assert.strictEqual(currentURL(), urls.user);
     assert.strictEqual(this.server.db.users[0].name, 'Updated user name');
```

Comment:
Not sure if this has been discussed before or not. Some tests files use this.server.db while others use this.server.schema. Overall they don't seem any different. Is there a preference of which one to use?

Reply:
I had actually asked about this before, it's probably better to use `schema` as that's what returns the actual models we use while DB is more of an internal implementation.